src/main/java/de/hysky/skyblocker/mixins/HandledScreenMixin.java

```diff
@@ -203,7 +203,7 @@ protected HandledScreenMixin(Text title) {
             return;
         }
         if (this.handler instanceof GenericContainerScreenHandler genericContainerScreenHandler && genericContainerScreenHandler.getRows() == 6) {
-            VisitorHelper.onSlotClick(slot, slotId, title);
+            VisitorHelper.onSlotClick(genericContainerScreenHandler.getSlot(13), slotId, title);
```

Comment:
I'm confused, clicking any slot clicks the 13th slot?

Reply:
I just needed to get the visitor head item (aka the 13th slot), and that slot parameter was unused, so I just thought why not use it

Reply:
skull emoji

Reply:
what? it's called recycling